<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainInterface</class>
 <widget class="QWidget" name="MainInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1094</width>
    <height>1146</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>-9</x>
     <y>1</y>
     <width>201</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="controlPanel"/>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>871</width>
     <height>461</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="graph">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="Chanderlier">
       <attribute name="title">
        <string>Evolution Cours</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_7">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>861</width>
          <height>431</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="chanderlierLayout"/>
       </widget>
      </widget>
      <widget class="QWidget" name="Bargraph">
       <attribute name="title">
        <string>Volume de trading</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_8">
        <property name="geometry">
         <rect>
          <x>-1</x>
          <y>-1</y>
          <width>861</width>
          <height>431</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="volumeLayout"/>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="addIndicateur">
   <property name="geometry">
    <rect>
     <x>221</x>
     <y>1</y>
     <width>184</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>EMA</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MACD</string>
    </property>
   </item>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>-9</x>
     <y>311</y>
     <width>201</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="history_transaction"/>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>500</y>
     <width>871</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="ROI"/>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>691</y>
     <width>811</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="Editeur_text"/>
  </widget>
  <widget class="QPushButton" name="pushButton_sauvegarder">
   <property name="geometry">
    <rect>
     <x>431</x>
     <y>1</y>
     <width>198</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Sauvegarder</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_retourMenu">
   <property name="geometry">
    <rect>
     <x>871</x>
     <y>1</y>
     <width>198</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Retour au menu</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_aide">
   <property name="geometry">
    <rect>
     <x>651</x>
     <y>1</y>
     <width>198</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Aide</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>821</x>
     <y>691</y>
     <width>251</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="listNote"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
